docs/source/onboarding_resources/computing/index.rst

```diff
@@ -0,0 +1,359 @@
+
+#. If you get tired of typing long commands, one option is to make an alias. An alias is a shortcut command for commonly typed things. More information on how to do was written by the `Cost Effectiveness team <https://hub.ihme.washington.edu/display/CE/Setting+up+cluster+access>`_ 
+#. When your computer falls asleep, it will stop access to the cluster and cut off any jobs that were currently running. This can be problematic if a notebook needs to run overnight. There are a few different options to account for this including: screen, MOSH, or tmux. If you need to use these, ask a teammate. 
+
+.. _cluster_access_files:
+
+File Systems and Storage 
+------------------------
+
+The cluster can be confusing with where to store code and data. Our 
+team has created some best practices to use for data storage. 
+
+For code, please create a new directory under :code:`/ihme/code` with your 
+username. For example, this might be :code:`/ihme/code/lutzes`. 
+You should clone GitHub respositories to this location and have 
+all Jupyter notebooks and other code stored here. 
+
+For data files, there are two locations based on the size of the 
+data file. 
+
+#. For small data files, store these on GitHub in the same location as your code. A "small" file is something without thousands of rows of data. Examples might include: a list of nicknames, disease severity proportions by age/sex group, or drug efficacy data. 
+#. For large files, store these in a shared location on the cluster. Considering making a new folder for each project for data storage. 
+#. When you deciding where to store data, please also consider any data restrictions that might exist. 
+
+Regardless of where you store data, it is important to track updates 
+to data files carefully. Engineers might copy and paste a file into a 
+new location, so updating the file might not actually change what is 
+being used in the sim. Therefore, follow these steps: 
+
+#. Use the naming conventions below to ensure consistency. 
+#. Always version up rather than replacing a data file (e.g., create a new file with the current date rather than just replacing with a different file of the same name). 
+#. Include the exact file name and location in the docs. This means if you version up a data file, you will need to update the docs to reflect the new name. This ensures the engineers are aware of any changes. 
+
+For consistency, please use this naming convention for all files: :code:`FILENAME_20230309.ext`. 
+For example, this might be :code:`heart_failure_proportions_20230310.csv` 
+
+.. _cluster_access_bash:
+
+Your Bash Files 
+---------------
+
+Bash files contain commands you would enter into a command line, but 
+specifically ones you will use very often. For example, every time you 
+want to open a Juptyer session, the cluster needs certain information 
+and requirements. To find this information, it looks in your Bash files 
+rather than asking you to enter the same information every time. 
```

Comment:
This is a bit inaccurate and potentially confusing. `.bash_profile` and `.bashrc` are scripts that run commands every time you start a new shell. It's by creating **aliases and functions** in these files that you are able to "save commands for later" by giving a short name to a common command.

Is there not a Hub page already that explains these files? I searched (useless, of course) and didn't find one. Happy to help write this part or look for non-IHME resources if we don't find anything. The Hub page linked below gives examples but does not explain how they work (and it seems like almost all of the examples don't apply to Sim Science).

Reply:
I just added this as a to-do for now since this PR is already giant and overwhelming!